pytest_doctestplus/plugin.py

```diff
@@ -33,7 +34,11 @@ def pytest_addoption(parser):
                      "plugin")
 
     parser.addoption("--doctest-rst", action="store_true",
-                     help="enable running doctests in .rst documentation")
+                     help="DEPRECATED, use doctest-glob. Enable running doctests in .rst documentation.")
```

Comment:
Maybe rewrite as ``Enable running doctests in .rst documentation. This is no longer recommended, use --doctest-glob="*.rst" instead.`` - I think let's hold off from calling this deprecated until we are ready to actually also raise deprecation warnings if the option is used etc. Let's have a phase where both are acceptable but glob is recommended, then next year we can deprecate the old options (in the mean time we can try and transition packages to using glob).